src/customMetadata/GS_Admin_Checklist_Item.Communicate_With_Users.md

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Communicate With Users</label>
+    <protected>true</protected>
+    <values>
+        <field>Description_Label__c</field>
+        <value xsi:type="xsd:string">gsChecklistItemCommunicateWithUserskDesc</value>
+    </values>
+    <values>
+        <field>Extra_Info_Label__c</field>
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>GS_Admin_Checklist_Section__c</field>
+        <value xsi:type="xsd:string">Bring_Users_Onboard</value>
+    </values>
+    <values>
+        <field>Has_Link__c</field>
+        <value xsi:type="xsd:boolean">true</value>
+    </values>
+    <values>
+        <field>Image__c</field>
+        <value xsi:type="xsd:string">Communicate_illustrations.svg</value>
```

Comment:
Do you need to specify the path for images - `gsAssets/step/Communicate_illustrations.svg`'?

Reply:
not only the name of the images in the folder gsAssets/step

Reply:
The helptext for the `Image__c` field in the CMDT object says "The path and filename of the image in the CumulusResources Static Resource to render." Is that not correct?

Reply:
A miss add in gAssets/step folder in the helper text

Reply:
So, you're confirming the that the path is not needed in this field? Which means that the LWC hardcodes the path for all images?

Reply:
Maybe is better "The path and filename of the image in the CumulusResources Static Resource under folder gsAssets/step  to render."

Reply:
@force2b

Reply:
Good suggestion. I'll update the google doc accordingly. FYI @dcaballeroSf